src/assets/translations/en/main.json

```diff
@@ -2974,6 +2974,7 @@
       "pending": "Pending",
       "claimAvailable": "Claim Available",
       "unknown": "Unknown"
-    }
+    },
+    "notificationTitle": "%{sellAmountAndSymbol} → %{buyAmountAndSymbol}"
```

Comment:
suggestion: this is only one specific kind of notification for swaps, title should match intent.
Maybe add a new `notificationCenter` key and make this `swapTitle`?